lua/telescope/_extensions/lazy/actions.lua

```diff
@@ -97,6 +97,9 @@ function M.open_in_browser()
     end
 
     local ret = vim.fn.jobstart({ open_cmd, selected_entry.url }, { detach = true })
+    if telescope_lazy_config.opts.open_in_browser_opts.close_window_after_action then
```

Comment:
Please move the action below the return code check, make sure this is separated with a newline from the check.